src/ipc/Kid.cc

```diff
@@ -23,6 +25,7 @@ Kid::Kid():
     badFailures(0),
     pid(-1),
     startTime(0),
+    stopTime(0),
```

Comment:
Please use the C++11 initialization style for new members. That will allow removal of the code duplication here and the next chunk.